test/conftest.py

```diff
@@ -262,7 +262,7 @@ def _create_autosubmit_exp(
         #       needed, especially if the disk has the valid value?
         config.experiment_data['DEFAULT']['EXPID'] = expid
 
-        autosubmit.create(expid, noplot=True, hide=False, force=True)
+        autosubmit.create(expid, noplot=True, hide=False, force=True, check_wrappers=True)
```

Comment:
While harmless, maybe it'd be safer to call `check_wrappers` only when we want to generate wrappers? We could add an argument to the fixture (I think it's what Pytest calls a "factory" fixture?).